rustfmt.toml

```
@@ -0,0 +1,4 @@
+imports_granularity = "Item"
```

Hot garbage. Use this and no less.
```suggestion
imports_granularity = "Crate"
```